src/plugins/preauth/pkinit/pkinit_crypto_openssl.c

```diff
@@ -5498,8 +5490,7 @@ load_cas_and_crls_dir(krb5_context context,
 
     while ((dentry = readdir(d))) {
         if (strlen(dirname) + strlen(dentry->d_name) + 2 > sizeof(filename)) {
-            pkiDebug("%s: Path too long -- directory '%s' and file '%s'\n",
-                     __FUNCTION__, dirname, dentry->d_name);
+            TRACE_PKINIT_PATH_TOO_LONG(context, dirname, dentry->d_name);
```

Comment:
This code should use asprintf(); in the meantime, I wouldn't add a trace message.